package.json

```diff
@@ -63,7 +63,7 @@
     "prop-types": "^15.8.0",
     "react-inlinesvg": "^2.3.0",
     "react-popper": "^2.2.5",
-    "react-select": "^3.1.0",
+    "react-select": "^4.0.0",
```

Comment:
please use the correct node version, run `nvm use` before and than `npm i`

Reply:
I have run it but did not understand what it did